Undefined names: 'score' and 'MediaDiscovererDescription'

Undefined names have the potential to raise NameError at runtime...

flake8 testing of https://github.com/Tribler/tribler on Python 2.7.14

$ __flake8 . --count --select=E901,E999,F821,F822,F823 --show-source --statistics__
```
./Tribler/Core/Modules/restapi/util.py:92:12: F821 undefined name 'score'
    return score
           ^
./TriblerGUI/vlc.py:4779:50: F821 undefined name 'MediaDiscovererDescription'
                   ctypes.POINTER(ctypes.POINTER(MediaDiscovererDescription)))
                                                 ^
./TriblerGUI/vlc.py:4791:41: F821 undefined name 'MediaDiscovererDescription'
                   None, ctypes.POINTER(MediaDiscovererDescription), ctypes.c_size_t)
                                        ^
3     F821 undefined name 'score'
3
```

--- a/Tribler/Core/Modules/restapi/util.py
+++ b/Tribler/Core/Modules/restapi/util.py
@@ -89,8 +89,6 @@ def get_parameter(parameters, name):
         return None
     return parameters[name][0]
 
-    return score
-
 
 def fix_unicode_dict(d):
     """

